web/pgadmin/browser/server_groups/servers/databases/publications/tests/15_plus/create_publication_msql.sql

```diff
@@ -0,0 +1,4 @@
+CREATE PUBLICATION test_publication_create
+   FOR ALL TABLES
+
```

Comment:
Why empty line?

Reply:
The sql empty line gets generated if all_tables switch is made to 'Yes' as the pubschema and pubtable will not be present in the data. Will fix try to fix this as well.

Reply:
Resolved